omod/src/test/java/org/openmrs/module/idgen/web/search/LogEntrySearchHandlerTest.java

```diff
@@ -0,0 +1,73 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.idgen.web.search;
+
+import org.junit.Test;
+import org.junit.Before;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.idgen.service.IdentifierSourceService;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.test.Util;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceControllerTest;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import org.junit.Assert;
+
+public class LogEntrySearchHandlerTest extends MainResourceControllerTest {
+    private static final String LOG_ENTRIES_XML = "logEntriesTestDataset.xml";
+
+    private IdentifierSourceService service;
+
```

Comment:
Can you look at the search handler test i pointed you in the ticket description and then put some thing in these method bodies?

Reply:
When I looked at the searchhandler tests, the methods looked quite similar to those I had in the controller. Is that the case?

Reply:
Just do them as in that search handler. Then we shall review.

Reply:
alright